sdk/schemaregistry/azure-data-schemaregistry-jsonschema/README.md

```diff
@@ -0,0 +1,163 @@
+
+## Getting started
+
+### Prerequisites
+
+- A [Java Development Kit (JDK)][jdk_link], version 8 or later.
+- [Azure Subscription][azure_subscription]
+- An [Event Hubs namespace][event_hubs_namespace]
+
+### Include the Package
+
+[//]: # ({x-version-update-start;com.azure:azure-data-schemaregistry-jsonschema;current})
+```xml
+<dependency>
+  <groupId>com.azure</groupId>
+  <artifactId>azure-data-schemaregistry-jsonschema</artifactId>
+  <version>1.0.0-beta.1</version>
+</dependency>
+```
+[//]: # ({x-version-update-end})
+
+### Create `SchemaRegistryJsonSchemaSerializer` instance
+
+The `SchemaRegistryJsonSchemaSerializer` instance is the main class that provides APIs for serializing and
+deserializing JSON schema format. The JSON schema is stored and retrieved from the Schema Registry service
+through the `SchemaRegistryAsyncClient`. So, before we create the serializer, we should create the client.
+
+#### Create `SchemaRegistryAsyncClient` with Azure Active Directory Credential
+
+In order to interact with the Azure Schema Registry service, you'll need to create an instance of the
+`SchemaRegistryAsyncClient` class through the `SchemaRegistryClientBuilder`. You will need the Schema Registry **endpoint**.
+
+You can authenticate with Azure Active Directory using the [Azure Identity library][azure_identity]. Note that regional
+endpoints do not support AAD authentication. Create a [custom subdomain][custom_subdomain] for your resource in order to
+use this type of authentication.
+
+To use the [DefaultAzureCredential][DefaultAzureCredential] provider shown below, or other credential providers provided
+with the Azure SDK, please include the `azure-identity` package:
+
+[//]: # ({x-version-update-start;com.azure:azure-identity;dependency})
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-identity</artifactId>
+    <version>1.10.0</version>
+</dependency>
+```
+
+You will also need to [register a new AAD application][register_aad_app] and [grant access][aad_grant_access] to
+ Schema Registry service.
+
+```java readme-sample-createSchemaRegistryAsyncClient
+TokenCredential tokenCredential = new DefaultAzureCredentialBuilder().build();
+
+// {schema-registry-endpoint} is the fully qualified namespace of the Event Hubs instance. It is usually
+// of the form "{your-namespace}.servicebus.windows.net"
+SchemaRegistryAsyncClient schemaRegistryAsyncClient = new SchemaRegistryClientBuilder()
+    .fullyQualifiedNamespace("{your-event-hubs-namespace}.servicebus.windows.net")
+    .credential(tokenCredential)
+    .buildAsyncClient();
+```
+
+#### Create `SchemaRegistryJsonSchemaSerializer` through the builder
+
+```java readme-sample-createSchemaRegistryJsonSchemaSerializer
+SchemaRegistryJsonSchemaSerializer serializer = new SchemaRegistryJsonSchemaSerializerBuilder()
+    .schemaRegistryClient(schemaRegistryAsyncClient)
```

Comment:
non-blocking API suggestion: it might be nicer to provide a method on `SchemaRegistryAsyncClient` to create a serializer instead of adding a builder.

E.g. we do this in ACR:

```java
ContainerRegistryClient client = getAsyncClient();
// ContainerRepository  is de-facto another client with its own service methods
ContainerRepository repository = client.getRepository(repositoryName);

// getProperties is a service method
ContainerRepositoryProperties properties = repository.getProperties();
```

https://github.com/Azure/azure-sdk-for-java/blob/1c28cc0a732f0a25f135cf06a3b8b4e5198a9d5f/sdk/containerregistry/azure-containers-containerregistry/src/samples/java/com/azure/containers/containerregistry/ContainerRegistryClientJavaDocSnippets.java#L94

Reply:
We wanted to be able to support different serialization (apache avro, json schema), which may pull in unwanted dependencies.

Reply:
If you wanted to reply to another comment on the serialization, then users don't need to install any additional dependencies - they can still use the default one that falls back to JacksonAdapter internally. But they MAY install additional dependencies.